CHANGELOG.md

```diff
@@ -0,0 +1,9 @@
+# Changelog
+
+We follow [Semantic Versioning](http://semver.org/) as a way of measuring stability of an update. This
+means we will never make a backwards-incompatible change within a major version of the project.
+
+## v3.0.0
```

Comment:
My theory was to follow semver in that fragmenting the codebase is a breaking change, so all four parts (core, u/ToR, u/transcribot, and u/ToR_archivist) get set to 3.0.0 and then incremented individually as we work on them.